packages/dashboard/src/customization/widgets/lineSymbol/plugin.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import LineWidgetComponent from './component';
+import LineSymbolIcon from './icon';
+import type { DashboardPlugin } from '~/customization/api';
+import { LineWidget } from '../types';
+
+export const linePlugin: DashboardPlugin = {
+  install: ({ registerWidget }) => {
+    registerWidget<LineWidget>('line', {
+      render: (widget) => <LineWidgetComponent {...widget} />,
+      componentLibrary: {
+        name: 'Line Symbol',
```

Comment:
does this fit under the icon? I feel like the text might be too large?

Reply:
it doesn't get cropped, but it pushes other symbols a little to the side

Reply:
This is similar to what we see for Timeline, except just more